guides/hosting/performance/lock-store.md

```diff
@@ -0,0 +1,21 @@
+# Lock store
+
+Shopware uses [Symfony's Lock component](https://symfony.com/doc/5.4/lock.html) to implement locking functionality.
+By default, Symfony will use a local lock store, this means in cluster setups the locking will only work on a single machine, therefore it is highly recommended using one of the [supported remote stores](https://symfony.com/doc/5.4/components/lock.html#available-stores).
```

Comment:
I think it would be more precise to say on multi-machine setups naive file locks *will* break the system.